src/Workspaces/CoreTest/WorkspaceTests/GeneralWorkspaceTests.cs

```diff
@@ -142,6 +142,30 @@ public void TestChangeDocumentSourceCodeKind_TryApplyChanges_Throws()
             Assert.Equal(WorkspacesResources.Changing_document_property_is_not_supported, e.Message);
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.Workspace)]
+        public void TestChangeDefaultNamespace_TryApplyChanges_Throws()
+        {
+            using var ws = new NoChangesAllowedWorkspace();
+            var projectId = ws.AddProject("TestProject", LanguageNames.CSharp).WithDefaultNamespace("OriginalName").Id;
+
+            var newName = "ChangedName";
+            var newSolution = ws.CurrentSolution.WithProjectDefaultNamespace(projectId, newName);
+
+            NotSupportedException e = null;
+            try
```

Comment:
This should be using the Assert.Throws() helper from xunit.

Reply:
(beats me why the rest aren't...)

Reply:
thanks - I have updated all the asserts in this test class